modules/core/CMakeLists.txt

```diff
@@ -60,6 +60,23 @@ if(CV_TRACE AND HAVE_ITT)
   add_definitions(-DOPENCV_WITH_ITT=1)
 endif()
 
+# https://github.com/opencv/opencv/issues/24145
+ocv_check_environment_variables(OPENCV_IPP_MEAN)
+option(OPENCV_IPP_MEAN "Enable IPP optimizations for mean (+200Kb in binary size)" OFF)
+if(OPENCV_IPP_MEAN)
+  add_definitions(-DOPENCV_IPP_MEAN=1)
```

Comment:
It is better to apply definition to the specific file:

```
ocv_append_source_file_compile_definitions(${CMAKE_CURRENT_SOURCE_DIR}/src/mean.dispatch.cpp "OPENCV_IPP_MEAN=1")
```

Reply:
Thank you for yor review, this is more  better fix. I fix with it.